libraries/typescript/Microsoft.Bot.Builder.Adapters.WeChat/src/accessTokenStorage.ts

```diff
@@ -0,0 +1,59 @@
+/**
+ * @module wechat
+ */
+/**
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { Storage, StoreItems } from 'botbuilder-core';
+import { WeChatAccessToken } from './weChatSchema';
+
+/**
+ * Storage provider for access token.
```

Comment:
>Storage provider for access token. [](start = 3, length = 34)

This class doesn't appear to exported so should be marked "@private" to prevent it from showing up in any documentation.